filer/management/commands/filer_check.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from django.core.files.storage import DefaultStorage
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.module_loading import import_string
+from django.utils.six.moves import input
+from django.utils.translation import ugettext_lazy as _
+from filer.settings import DEFAULT_FILER_STORAGES
+
+
+class Command(BaseCommand):
```

Comment:
management commands shouldn't use mark strings for translation.

Reply:
fixed